spec/openapi.yaml

```diff
@@ -33,15 +33,21 @@ paths:
       summary: Create Publisher Agreement
       description: Create an open-vsx.org publisher agreement for the current user.
       requestBody:
-        description: The version number to update or insert for the current logged in user
+        description: Required information to sign an open-vsx.org publisher aggrement.
         content:
           application/json:
             schema:
               type: object
               properties:
-                Version:
+                version:
                   type: integer
```

Comment:
This should be a string to allow arbitrary version ids.

Reply:
Done!